src/server/middleware/csp.js

```diff
@@ -58,7 +59,17 @@ export function getCSP() {
 const csp = () => (req, res, next) => {
   const { policy } = cspCache;
   const scriptNonce = uuidV4();
-  const updatedPolicy = insertSource(policy, 'script-src', `'nonce-${scriptNonce}'`);
+  let updatedPolicy;
+  if (process.env.NODE_ENV === 'development') {
+    updatedPolicy = insertSource(
```

Comment:
#nitpick maybe separate the two `insertSource`s so they are not nested for readability

Reply:
separated in 0cee717